tsl/test/sql/CMakeLists.txt

```diff
@@ -115,6 +115,11 @@ if (PG_VERSION_SUPPORTS_MULTINODE)
       dist_hypertable_with_oids.sql
     )
   endif()
+  if (${PG_VERSION_MAJOR} GREATER_EQUAL "12")
```

Comment:
This should be part of TEST_FILES instead of TEST_FILES_DEBUG so that is always run? Don't think we have debug specific functionality here. Also could you rename the test file to something more descriptive and remove the version suffix  (like dist_hyertable_am.sql) ?

Reply:
Apparently all multinode tests are run only in DEBUG, not sure that it would be a good idea to make an exception here

Reply:
Multinode tests run in debug because most of the rely on `remote_exec`, which is only available in debug builds. I guess this PR's tests uses this function too, so need to be under debug.

We might want to consider coming up with a way of dynamically loading test functions into a release build or something. (FWIW, there's a story for running more tests in release builds).

Reply:
We could put these functions into a separate extension so that they are completely independent on the type of build.